File: Sorting/bubblesort.py
def BUBBLE_SORT(A):
    Start=0
    End=len(A)-1
    while(End!=Start):
        i=Start
        while(i<End):
            if(A[i+1]<A[i]):
                temp=A[i]
                A[i]=A[i+1]
                A[i+1]=temp
            i=i+1
        End=End-1
    return A

A=[4,5,8,1,3,6,2,7]
print(BUBBLE_SORT(A))

File: Sorting/bubble-sort.js
function bubbleSort(values) {
    
        var origValues = values.slice();
        var length = origValues.length - 1;
        do {
          var swapped = false;
          for(var i = 0; i < length; ++i) {
            if (origValues[i] > origValues[i+1]) {
              var temp = origValues[i];
              origValues[i] = origValues[i+1];
              origValues[i+1] = temp;
              swapped = true;
            }
          }
        }
        while(swapped === true);
        return origValues
      
}